src/controller/tag-controller.js

```diff
@@ -0,0 +1,40 @@
+import TagService from '../service/tag-service.js';
+
+class TagController {
+  async getTags(req, res, next) {
+    try {
+      const { page = 1, limit = 10, order = 'desc', search } = req.query;
+      const tags = await TagService.getTags(
```

Comment:
```suggestion
      const { tags, totalCount } = await TagService.getTags(
```
총 개수를 반환 받기 위해 작성한 코드입니다.